src/stylesheets/layouts/_concept.scss

```diff
@@ -0,0 +1,380 @@
+.solution {
+  background: linear-gradient($color-medium-blue 5%, $color-mint 90%);
+  @include flex;
+
+  &__wrapper {
+    position: relative;
+    @include flex;
+    width: 100%;
+    height: 100%;
+    max-width: 77rem;
+    margin: 0;
+    padding: 2.5rem 1.5rem;
+    @media (min-width: 550px) {
+      margin: 2rem;
+      padding: 4rem 6.5rem;
+    }
+    @media (min-width: 800px) {
+      margin: 2rem 2rem 0;
+      padding: 5rem 8.5rem 0;
+    }
+  }
+
+  &__florence {
+    width: 200px;
+    margin-top: -30%;
+    margin-bottom: 30px;
+  }
+
+  &__hdr {
+    margin-bottom: $content-margin-btm;
+
+    &--top {
+      display: block;
+      padding-bottom: $hdr-padding-btm;
+    }
+
+    &--btm {}
+  }
+
+  &__context {
+    margin-bottom: $content-margin-btm;
+  }
+
+  &__section {
+    &--1,
+    &--2,
+    &--3,
+    &--4,
+    &--5,
+    &--6,
+    &--7 {
+      position: relative;
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      width: 100%;
+      background-color: $color-white;
+      margin: 7rem 0;
+      padding: 5rem 1.5rem;
+      @media (min-width: 800px) {
+        flex-direction: row;
+        margin: 5rem;
+        max-width: 77rem;
+        background-color: $color-white;
+        min-height: 39.5rem;
+        padding: 5rem 8.5rem;
+        border-radius: 5px;
+      }
+    }
+
+    &--1 {}
+
+    &--2 {
+      @media (min-width: 800px) {
+        padding: 1rem 8.5rem;
+      }
+    }
+
+    &--6 {
+      @media (min-width: 800px) {
+        overflow: hidden;
+      }
+    }
+
+    &__left {
+      display: flex;
+      margin: 0;
+      flex-direction: column;
+      @media (min-width: 800px) {
+        flex-basis: 50%;
+        margin-right: 2rem;
+      }
+    }
+
+    &__right {
+      flex-basis: 40%;
+      margin-left: 4rem;
+      z-index: 2;
+    }
+
+    &__hdr {
+      padding-bottom: $hdr-padding-btm;
+    }
+
+    &__content {
+      margin-bottom: $content-margin-btm;
+      z-index: 2;
+    }
+
+    &__bg {
+      position: absolute;
+      z-index: 0;
+    }
+
+    &__img {
+      position: absolute;
+      z-index: 1;
+    }
+
+    &__img1-bg {
+      align-self: center;
+      width: 220px;
+      top: 0;
+      transform: rotate(-180deg);
+      @media (min-width: 550px) {}
+      @media (min-width: 800px) {
+        width: 300px;
+        top: 75%;
+        transform: rotate(360deg);
+      }
+    }
+
+    &__img1 {
+      align-self: center;
+      width: 145px;
+      top: -15%;
+      @media (min-width: 550px) {}
+      @media (min-width: 800px) {
+        width: 170px;
+        top: 71%;
+      }
```

Comment:
spaaaace. And you don't need that 550 breakpoint

Reply:
resolved in pr#4